tutorial_boosting.md

```diff
@@ -64,8 +68,12 @@ python search_with_boosting.py
 
 Now you can check ```results[]```, the products that correspond to the boost condition became reranked.
```

Comment:
Suggestion:
Now you can check ```results[]```. The products corresponding to the boost condition became reranked.